S32K/libuavcan/media/S32K/canfd.hpp

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (c) 2020, NXP. All rights reserved.
+ * Distributed under The MIT License.
+ * Author: Abraham Rodriguez <abraham.rodriguez@nxp.com>
+ */
+
+/**
+ * @file
+ * Header driver file for the media layer of Libuavcan v1 targeting
+ * the NXP S32K14 family of automotive grade MCU's running
+ * CAN-FD at 4Mbit/s data phase and 1Mbit/s in nominal phase.
+ */
+
+#ifndef CANFD_HPP_INCLUDED
+#define CANFD_HPP_INCLUDED
+
+/** Driver build configurations file */
+#include "build_config.hpp"
+
+/** libuavcan core header files */
+#include "libuavcan/media/can.hpp"
+#include "libuavcan/media/interfaces.hpp"
+#include "libuavcan/platform/memory.hpp"
+
+/** STL queue for the intermediate ISR buffer */
+#include <deque>
```

Comment:
move this to the .cpp

Reply:
Implemented at https://github.com/noxuz/platform_specific_components/commit/f86edb519ec3bd86aff1aade093e4e3084f4c009